tuner/tuner/common.py

```diff
@@ -70,32 +66,59 @@ def __str__(self) -> str:
         return "x".join(map(dim_to_str, self.shape)) + "x" + str(self.element_type)
 
 
+# Represents the size of the iteration space along each contraction dimension.
+# For example, the following is a simple batch mmt:
+#   linalg.generic ... indexing_maps = [
+#       affine_map<(b, m, n, k) -> (b, m, k)>,
+#       affine_map<(b, m, n, k) -> (b, n, k)>,
+#       affine_map<(b, m, n, k) -> (b, m, n)>,
+#     ] ...
+#     ins(%lhs: tensor<4x8x32xf16>, %rhs: tensor<4x16x32xf16>)
+#     outs(%acc: tensor<4x8x16xf16>)
+# The ContractionSizes would be:
+#   M = [8]
+#   N = [16]
+#   K = [32]
+#   B = [4]
```

Comment:
Can you make this a docstring? https://peps.python.org/pep-0257/#multi-line-docstrings